Cql/Cql.Operators/CqlOperators.cs

```diff
@@ -171,10 +171,13 @@ public T Message<T>(T source, string code, string severity, string message)
         public IEnumerable<T>? Where<T>(IEnumerable<T>? source, Func<T, bool?> lambda) =>
             source == null ? null : source.Where(x => lambda(x) ?? false).ToList();
 
-        public TAccumulate? Aggregate<TSource, TAccumulate>(IEnumerable<TSource?>? source, TAccumulate? seed, Func<TAccumulate?, TSource?, TAccumulate?> lambda) =>
+        public TAccumulate? Aggregate<TSource, TAccumulate>(
+            IEnumerable<TSource?>? source,
+            Func<TAccumulate?, TSource?, TAccumulate?> lambda,
+            TAccumulate? seed) =>
             source == null ? default : source.Aggregate(seed, lambda);
 
-        public IValueSetFacade CreateValueSetFacade(CqlValueSet valueSet) => ValueSets.GetValueSet(valueSet);
+        public IValueSetFacade ExpandValueSet(CqlValueSet valueSet) => ValueSets.GetValueSet(valueSet);

```

Comment:
Ok, then this would have to be an exception on the name matching between the Elm.ExpandValueSet and the ICqlOperator method for it

Reply:
It could be it is a real operation - and since the current implementation of the valueset system *always* supplied expanded valuesets, Evan decided to hardwire the operation to just returning the valueset because of that....